streams/ru.m3u

```diff
@@ -805,3 +805,9 @@ http://live.ugratv.cdnvideo.ru/ugratv/smil:ugrastream1.smil/playlist.m3u8
 http://yurgan.bonus-tv.ru/cdn/yurgan/playlist.m3u8
 #EXTINF:-1 tvg-id="YuTV.ru" status="online",ЮТВ (Чебоксары) (432p) [Not 24/7]
 http://serv24.vintera.tv:8081/utv/Stream/playlist.m3u8
+#EXTINF:-1 tvg-id="RBKTV.ru" status="online",РБК (1080p)
+https://live-rbc.gcdn.co/gcore/rbctvhd_1080p/index.m3u8
+#EXTINF:-1 tvg-id="RBKTV.ru" status="online",РБК (720p)
+https://live-rbc.gcdn.co/gcore/rbctvhd_720p/index.m3u8
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="RBKTV.ru" status="online",РБК (1080p)
https://live-rbc.gcdn.co/gcore/rbctvhd.m3u8
```
Adaptive

Reply:
Thx for the find
But apparently I shouldn't have pressed that "commit suggestion" button personally 🙄